Update few `fast/multicol` tests to 'reftests'

Ahmad-S792 opened the pull request:
#### a07a86fa10d6b68236ff26605e878e3a39355668
<pre>
Update few `fast/multicol` tests to &apos;reftests&apos;

<a href="https://bugs.webkit.org/show_bug.cgi?id=275879">https://bugs.webkit.org/show_bug.cgi?id=275879</a>

Reviewed by Alan Baradlay.

Merge: <a href="https://github.com/chromium/chromium/commit/67008eb36304e282288cd2d52b0634bc3f9c15c5">https://github.com/chromium/chromium/commit/67008eb36304e282288cd2d52b0634bc3f9c15c5</a> ,
<a href="https://github.com/chromium/chromium/commit/fb77f07940e4162728766c9086d5859031ba31b8">https://github.com/chromium/chromium/commit/fb77f07940e4162728766c9086d5859031ba31b8</a> &amp;
<a href="https://github.com/chromium/chromium/commit/1eade67072746a496d824e9387c4f3e28132b431">https://github.com/chromium/chromium/commit/1eade67072746a496d824e9387c4f3e28132b431</a>

* LayoutTests/fast/multicol/column-count-with-rules-expected.html: Added.
* LayoutTests/fast/multicol/column-count-with-rules.html:
* LayoutTests/fast/multicol/nested-columns-expected.html: Added.
* LayoutTests/fast/multicol/vertical-lr/column-count-with-rules-expected.html: Added.
* LayoutTests/fast/multicol/vertical-lr/column-count-with-rules.html:
* LayoutTests/fast/multicol/vertical-lr/nested-columns-expected.html: Added.
* LayoutTests/fast/multicol/vertical-rl/column-count-with-rules-expected.html: Added.
* LayoutTests/fast/multicol/vertical-rl/column-count-with-rules.html:
* LayoutTests/fast/multicol/vertical-rl/nested-columns-expected.html: Added.
* LayoutTests/fast/multicol/vertical-rl/rule-style-expected.html: Added.
* LayoutTests/fast/multicol/vertical-rl/rule-style-expected.txt: Removed.
* LayoutTests/fast/multicol/vertical-rl/rule-style.html:
* LayoutTests/platform/glib/fast/multicol/column-count-with-rules-expected.txt: Removed.
* LayoutTests/platform/glib/fast/multicol/nested-columns-expected.txt: Removed.
* LayoutTests/platform/glib/fast/multicol/vertical-lr/column-count-with-rules-expected.txt: Removed.
* LayoutTests/platform/glib/fast/multicol/vertical-lr/nested-columns-expected.txt: Removed.
* LayoutTests/platform/glib/fast/multicol/vertical-rl/column-count-with-rules-expected.txt: Removed.
* LayoutTests/platform/glib/fast/multicol/vertical-rl/nested-columns-expected.txt: Removed.
* LayoutTests/platform/glib/fast/multicol/vertical-rl/rule-style-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/multicol/column-count-with-rules-expected.png: Removed.
* LayoutTests/platform/gtk/fast/multicol/nested-columns-expected.png: Removed.
* LayoutTests/platform/gtk/fast/multicol/vertical-lr/column-count-with-rules-expected.png: Removed.
* LayoutTests/platform/gtk/fast/multicol/vertical-lr/nested-columns-expected.png: Removed.
* LayoutTests/platform/gtk/fast/multicol/vertical-rl/column-count-with-rules-expected.png: Removed.
* LayoutTests/platform/gtk/fast/multicol/vertical-rl/nested-columns-expected.png: Removed.
* LayoutTests/platform/gtk/fast/multicol/vertical-rl/rule-style-expected.png: Removed.
* LayoutTests/platform/ios/fast/multicol/column-count-with-rules-expected.txt: Removed.
* LayoutTests/platform/ios/fast/multicol/nested-columns-expected.txt: Removed.
* LayoutTests/platform/ios/fast/multicol/unsplittable-inline-block-expected.txt: Removed.
* LayoutTests/platform/ios/fast/multicol/vertical-lr/column-count-with-rules-expected.txt: Removed.
* LayoutTests/platform/ios/fast/multicol/vertical-lr/nested-columns-expected.txt: Removed.
* LayoutTests/platform/ios/fast/multicol/vertical-lr/unsplittable-inline-block-expected.txt: Removed.
* LayoutTests/platform/ios/fast/multicol/vertical-rl/column-count-with-rules-expected.txt: Removed.
* LayoutTests/platform/ios/fast/multicol/vertical-rl/nested-columns-expected.txt: Removed.
* LayoutTests/platform/ios/fast/multicol/vertical-rl/unsplittable-inline-block-expected.txt: Removed.
* LayoutTests/platform/mac/fast/multicol/column-count-with-rules-expected.png: Removed.
* LayoutTests/platform/mac/fast/multicol/column-count-with-rules-expected.txt: Removed.
* LayoutTests/platform/mac/fast/multicol/nested-columns-expected.png: Removed.
* LayoutTests/platform/mac/fast/multicol/nested-columns-expected.txt: Removed.
* LayoutTests/platform/mac/fast/multicol/vertical-lr/column-count-with-rules-expected.png: Removed.
* LayoutTests/platform/mac/fast/multicol/vertical-lr/column-count-with-rules-expected.txt: Removed.
* LayoutTests/platform/mac/fast/multicol/vertical-lr/nested-columns-expected.png: Removed.
* LayoutTests/platform/mac/fast/multicol/vertical-lr/nested-columns-expected.txt: Removed.
* LayoutTests/platform/mac/fast/multicol/vertical-rl/column-count-with-rules-expected.png: Removed.
* LayoutTests/platform/mac/fast/multicol/vertical-rl/column-count-with-rules-expected.txt: Removed.
* LayoutTests/platform/mac/fast/multicol/vertical-rl/nested-columns-expected.png: Removed.
* LayoutTests/platform/mac/fast/multicol/vertical-rl/nested-columns-expected.txt: Removed.
* LayoutTests/platform/mac/fast/multicol/vertical-rl/rule-style-expected.png: Removed.
* LayoutTests/platform/wincairo/fast/multicol/column-count-with-rules-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/multicol/nested-columns-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/multicol/vertical-lr/column-count-with-rules-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/multicol/vertical-lr/nested-columns-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/multicol/vertical-rl/column-count-with-rules-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/multicol/vertical-rl/nested-columns-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/multicol/vertical-rl/rule-style-expected.png: Removed.
* LayoutTests/fast/multicol/nested-columns.html:
* LayoutTests/fast/multicol/vertical-lr/nested-columns.html:
* LayoutTests/fast/multicol/vertical-rl/nested-columns.html:

Canonical link: <a href="https://commits.webkit.org/280375@main">https://commits.webkit.org/280375@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/c11f39acf23c1cd75c5cff3a210facd40a1ec82f

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/56454 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/35780 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/8926 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/60061 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/59/builds/6890 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/43402 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/7084 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/45728 "Passed tests") | [✅ 🧪 wincairo-tests](https://ews-build.webkit.org/#/builders/60/builds/4820 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/58483 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/33646 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/48718 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/26590 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/30423 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/6043 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/5894 "Built successfully") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/52400 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/6315 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/61745 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/87/builds/362 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/62/builds/6429 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/52989 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/86/builds/362 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/48779 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/52883 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/12497 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/88/builds/322 "Passed tests") | | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/31607 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/32693 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/33776 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/32440 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

alanbaradlay commented on LayoutTests/fast/multicol/vertical-rl/nested-columns.html:
```diff
@@ -1,18 +1,13 @@
-<div style="-webkit-column-count:2">
-Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Nulla varius enim ac mi. Curabitur sollicitudin felis quis lectus. Quisque adipiscing rhoncus sem. Proin nulla purus, vulputate vel, varius ut, euismod et, nisi. Sed vitae felis vel orci sagittis aliquam. Cras convallis adipiscing sem. Nam nonummy enim. Nullam bibendum lobortis neque. Vestibulum velit orci, tempus euismod, pretium quis, in ...
-Suspendisse hendrerit quam nec felis. Sed varius turpis vitae pede. <span style="position:relative; opacity:0.5">Lorem ipsum dolor sit amet, consectetuer adipiscing elit.</span>
+<!DOCTYPE html>
+<p>The word 'PASSED' should be seen below (writing-mode is vertical-rl).</p>
+<div style="width:2em; height:6em; -webkit-columns:3; -webkit-column-gap:0; column-fill:auto; line-height:2em; -webkit-writing-mode:vertical-rl;">
+    <div style="-webkit-column-count:2; -webkit-column-gap:0;">
+        P<br>A
+    </div>
+    <div style="-webkit-column-count:2; -webkit-column-gap:0;">
+        S<br>S
+    </div>
+    <div style="-webkit-column-count:2; -webkit-column-gap:0;">
+        E<br>D
+    </div>
```
I am seeing lots of red in .html files. how is this just a reftest update? (I must be missing something here)

Ahmad-S792 replied:
So Blink tweaked 'test' to get the same behavior in order to make them 'reftest' without adding too much text etc.

Ahmad-S792 replied:
@alanbaradlay - same as this - https://github.com/chromium/chromium/commit/fb77f07940e4162728766c9086d5859031ba31b8

alanbaradlay replied:
and we are sure about not losing coverage here.

Ahmad-S792 replied:
Yes - same test still exists on Blink as well. :-)

Commit:
Update few `fast/multicol` tests to 'reftests'

https://bugs.webkit.org/show_bug.cgi?id=275879

Reviewed by Alan Baradlay.

Merge: https://github.com/chromium/chromium/commit/67008eb36304e282288cd2d52b0634bc3f9c15c5 ,
https://github.com/chromium/chromium/commit/fb77f07940e4162728766c9086d5859031ba31b8 &
https://github.com/chromium/chromium/commit/1eade67072746a496d824e9387c4f3e28132b431

* LayoutTests/fast/multicol/column-count-with-rules-expected.html: Added.
* LayoutTests/fast/multicol/column-count-with-rules.html:
* LayoutTests/fast/multicol/nested-columns-expected.html: Added.
* LayoutTests/fast/multicol/vertical-lr/column-count-with-rules-expected.html: Added.
* LayoutTests/fast/multicol/vertical-lr/column-count-with-rules.html:
* LayoutTests/fast/multicol/vertical-lr/nested-columns-expected.html: Added.
* LayoutTests/fast/multicol/vertical-rl/column-count-with-rules-expected.html: Added.
* LayoutTests/fast/multicol/vertical-rl/column-count-with-rules.html:
* LayoutTests/fast/multicol/vertical-rl/nested-columns-expected.html: Added.
* LayoutTests/fast/multicol/vertical-rl/rule-style-expected.html: Added.
* LayoutTests/fast/multicol/vertical-rl/rule-style-expected.txt: Removed.
* LayoutTests/fast/multicol/vertical-rl/rule-style.html:
* LayoutTests/platform/glib/fast/multicol/column-count-with-rules-expected.txt: Removed.
* LayoutTests/platform/glib/fast/multicol/nested-columns-expected.txt: Removed.
* LayoutTests/platform/glib/fast/multicol/vertical-lr/column-count-with-rules-expected.txt: Removed.
* LayoutTests/platform/glib/fast/multicol/vertical-lr/nested-columns-expected.txt: Removed.
* LayoutTests/platform/glib/fast/multicol/vertical-rl/column-count-with-rules-expected.txt: Removed.
* LayoutTests/platform/glib/fast/multicol/vertical-rl/nested-columns-expected.txt: Removed.
* LayoutTests/platform/glib/fast/multicol/vertical-rl/rule-style-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/multicol/column-count-with-rules-expected.png: Removed.
* LayoutTests/platform/gtk/fast/multicol/nested-columns-expected.png: Removed.
* LayoutTests/platform/gtk/fast/multicol/vertical-lr/column-count-with-rules-expected.png: Removed.
* LayoutTests/platform/gtk/fast/multicol/vertical-lr/nested-columns-expected.png: Removed.
* LayoutTests/platform/gtk/fast/multicol/vertical-rl/column-count-with-rules-expected.png: Removed.
* LayoutTests/platform/gtk/fast/multicol/vertical-rl/nested-columns-expected.png: Removed.
* LayoutTests/platform/gtk/fast/multicol/vertical-rl/rule-style-expected.png: Removed.
* LayoutTests/platform/ios/fast/multicol/column-count-with-rules-expected.txt: Removed.
* LayoutTests/platform/ios/fast/multicol/nested-columns-expected.txt: Removed.
* LayoutTests/platform/ios/fast/multicol/unsplittable-inline-block-expected.txt: Removed.
* LayoutTests/platform/ios/fast/multicol/vertical-lr/column-count-with-rules-expected.txt: Removed.
* LayoutTests/platform/ios/fast/multicol/vertical-lr/nested-columns-expected.txt: Removed.
* LayoutTests/platform/ios/fast/multicol/vertical-lr/unsplittable-inline-block-expected.txt: Removed.
* LayoutTests/platform/ios/fast/multicol/vertical-rl/column-count-with-rules-expected.txt: Removed.
* LayoutTests/platform/ios/fast/multicol/vertical-rl/nested-columns-expected.txt: Removed.
* LayoutTests/platform/ios/fast/multicol/vertical-rl/unsplittable-inline-block-expected.txt: Removed.
* LayoutTests/platform/mac/fast/multicol/column-count-with-rules-expected.png: Removed.
* LayoutTests/platform/mac/fast/multicol/column-count-with-rules-expected.txt: Removed.
* LayoutTests/platform/mac/fast/multicol/nested-columns-expected.png: Removed.
* LayoutTests/platform/mac/fast/multicol/nested-columns-expected.txt: Removed.
* LayoutTests/platform/mac/fast/multicol/vertical-lr/column-count-with-rules-expected.png: Removed.
* LayoutTests/platform/mac/fast/multicol/vertical-lr/column-count-with-rules-expected.txt: Removed.
* LayoutTests/platform/mac/fast/multicol/vertical-lr/nested-columns-expected.png: Removed.
* LayoutTests/platform/mac/fast/multicol/vertical-lr/nested-columns-expected.txt: Removed.
* LayoutTests/platform/mac/fast/multicol/vertical-rl/column-count-with-rules-expected.png: Removed.
* LayoutTests/platform/mac/fast/multicol/vertical-rl/column-count-with-rules-expected.txt: Removed.
* LayoutTests/platform/mac/fast/multicol/vertical-rl/nested-columns-expected.png: Removed.
* LayoutTests/platform/mac/fast/multicol/vertical-rl/nested-columns-expected.txt: Removed.
* LayoutTests/platform/mac/fast/multicol/vertical-rl/rule-style-expected.png: Removed.
* LayoutTests/platform/wincairo/fast/multicol/column-count-with-rules-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/multicol/nested-columns-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/multicol/vertical-lr/column-count-with-rules-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/multicol/vertical-lr/nested-columns-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/multicol/vertical-rl/column-count-with-rules-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/multicol/vertical-rl/nested-columns-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/multicol/vertical-rl/rule-style-expected.png: Removed.
* LayoutTests/fast/multicol/nested-columns.html:
* LayoutTests/fast/multicol/vertical-lr/nested-columns.html:
* LayoutTests/fast/multicol/vertical-rl/nested-columns.html:

Canonical link: https://commits.webkit.org/280375@main
